drivers/dht/include/dht_params.h

```diff
@@ -34,11 +34,15 @@ extern "C" {
 #ifndef DHT_PARAM_PIN
 #define DHT_PARAM_PIN               (GPIO_PIN(0, 0))
 #endif
+#ifndef DHT_PARAM_TYPE
+#define DHT_PARAM_TYPE              (DHT11)
+#endif
 #ifndef DHT_PARAM_PULL
-#define DHT_PARAM_PULL              (GPIO_IN_PU)
+#define DHT_PARAM_PULL              (GPIO_IN)
```

Comment:
Why?

What is the reason to default to no PU?

Reply:
Firstly, because the DHT already has an internal pull-up resistor, it is not necessary to unnecessarily increase power consumption. Additionally, the nominal voltage of DHT is 5V, and some 3.3V MCUs have their I/Os tolerant to 5V, provided that the internal pull-up resistor is not activated.

Reply:
so we can remove the runtime `<dht_params_t>.in_mode` information since it is mcu/board specific and not likely to change at runtime -> we can use the define. 

you may ignore this